cfme/intelligence/chargeback/rates.py

```diff
@@ -57,6 +57,9 @@ class fields(ParametrizedView):  # noqa
         PARAMETERS = ('name',)
         ROOT = ParametrizedLocator('.//tr[./td[contains(normalize-space(.), {name|quote})]]')
 
+        def __locator__(self):
+            return self.browser.element(self.ROOT, parent=self.parent)
```

Comment:
REQUIRED: Returning elements in locators is strictly prohibited unless there is no other option, which in 99% of cases can be worked around.

Besides, ROOT automatically generates locator which does roughly what this unnecessary method does.